### PR TITLE
refactor: organize projects/managing.md

### PR DESCRIPTION
**Pull request by IgnisRBX**
## Checks

By submitting your pull request for review, you agree to the following:

- [ ] This contribution was created in whole or in part by me, and I have the right to submit it under the terms of this repository's open source licenses.
- [ ] I understand and agree that this contribution and a record of it are public, maintained indefinitely, and may be redistributed under the terms of this repository's open source licenses.
- [ ] To the best of my knowledge, all proposed changes are accurate.

### CODE REVIEW

File: content/en-us/projects/managing.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ description: An overview of managing projects with Roblox Studio.
 To create a project:
 
 1. Open [Roblox Studio](../studio/setting-up-roblox-studio.md).
-2. Click a template from the Studio start screen. It's highly recommended to select a [streaming](../workspace/streaming.md) template, which improves performance by dynamically loading and unloading 3D content and related instances in regions of the world.
+2. Click a template from the Studio start screen.
```
Comment (IgnisRBX) [inline content/en-us/projects/managing.md:11]: All Studio templates are streaming-enabled now... no need to choose either way.

File: content/en-us/projects/managing.md [hunk 2]
```diff
@@ -33,6 +33,37 @@ define project settings:
    This unlocks more tabs to define other project settings as
    described in [Game Settings](../studio/game-settings.md).
 
+## Cloud-Based Assets
```
Comment (IgnisRBX) [inline content/en-us/projects/managing.md:36]: Just moving these two sections up from a bit lower down. Reading about assets and processes for testing naturally come before publishing in the typical workflow.

File: content/en-us/projects/managing.md [hunk 3]
```diff
@@ -179,7 +177,7 @@ Modeling](/art/modeling).
 
 <img src="../assets/misc/Building-Best-Practices-Example-Parts.jpg" />
 
-### Default Materials
+### Applying Materials
```
Comment (IgnisRBX) [inline content/en-us/projects/managing.md:180]: This is more about material usage in general, including custom materials, not just the defaults.

File: content/en-us/projects/managing.md [hunk 4]
```diff
@@ -229,49 +227,6 @@ Showcase](/resources/beyond-the-dark) and [The Mystery of Duvall Drive](../resou
 
 <img src="../assets/resources/beyond-the-dark/building-architecture/Architecture-Overview.jpeg" />
 
-## Optimization and Performance
```
Comment (IgnisRBX) [inline content/en-us/projects/managing.md:232]: All of this is outdated and has been replaced by the new, dedicated "Performance Optimization" articles also in the Projects category.

File: content/en-us/projects/managing.md [hunk 5]
```diff
@@ -33,6 +33,37 @@ define project settings:
...
+  running on the Roblox application.
+- **Multi-client simulation** for comparing how each client "sees" other clients
+  within the experience.
+- **Device emulation** that provides insight on how controls operate on a mobile
+  device or how on-screen UI displays on different screens and aspect ratios.
+- **Collaborative playtesting** with members of your
+  [team](../projects/setting-up-a-team.md).
+
+For more information on each testing option, see [Studio Testing
+Modes](/studio/testing-modes).
```
Comment (aetter) [inline content/en-us/projects/managing.md:65]: Convert to relative, add `.md`?

File: content/en-us/projects/managing.md [hunk 6]
```diff
@@ -33,6 +33,37 @@ define project settings:
...
+experience. Each asset in the cloud is assigned a unique **asset ID** from which
+multiple experiences can utilize them.
+
+See [Assets](../projects/assets/index.md) for details on instances, assets,
+and their association in Roblox.
+
+## Testing
+
+Your team can instantly test the experience on PC, mobile, and other devices you
+wish to support through the Roblox app — no need to compile builds, deploy to
```
Comment (aetter) [inline content/en-us/projects/managing.md:49]:
```suggestion
wish to support through the Roblox app—no need to compile builds, deploy to
```

**Review by aetter — APPROVED** [review]

File: content/en-us/projects/managing.md (hunk 5 above)
Comment (IgnisRBX) [inline content/en-us/projects/managing.md:65]:
```suggestion
Modes](../studio/testing-modes.md).
```

**Comment by IgnisRBX** [comment]
@tchu-roblox Not sure why this needs another review, since Andrew already approved it, but could you please do so? Thanks!

**Pull request merged**